src/python/cryptography_silent_warnings.patch

```diff
@@ -0,0 +1,21 @@
+--- Lib/site.py.orig    2020-11-12 09:59:21.576077736 +0000
++++ Lib/site.py 2020-11-12 10:00:01.732005931 +0000
+@@ -63,6 +63,18 @@
+ import __builtin__
+ import traceback
+
++import warnings
++warnings.filterwarnings(
++    'ignore',
++    category=UserWarning,
++    module='OpenSSL.crypto',
++    )
++warnings.filterwarnings(
++    'ignore',
++    category=UserWarning,
++    module='cryptography.hazmat.bindings.openssl.binding',
++    )
```

Comment:
I found another way to ignore these warnings....and I think that this is the better approach

```
warnings.filterwarnings(
    'ignore',
    category=UserWarning,
    message='EXACT MESSAGE TO BE IGNORED',
    )
```

We should ignore both py 2.7 and openssl 1.0.2

This is better than filtering based on module, since the cryptography error is raised based on whichever other module is importing it.

Thanks

Reply:
Thanks, that would be much better. Going to give it a try…